src/apps/nem/mosaic/serialize.py

```diff
@@ -98,8 +94,10 @@ def _write_property(w: bytearray, name: str, value):
             value = "false"
     elif type(value) == int:
         value = str(value)
-    elif type(value) != str:
+    if type(value) != str:
```

Comment:
I think this is a bug. The `elif` should not be dropped

Reply:
The rest of the function depends on `value` being `str`, it was a bit clearer for me this way. But I don't really have any strong preference.

Reply:
Oh, now I got it. It's fine, I haven't realised all the "conversions" are to `str`.